docs/how-tos/configure.md

```diff
@@ -82,7 +87,7 @@ For example, if you have a DC motor, follow the [corresponding configuration ins
 
 <!-- markdownlint-disable MD036 -->
 
-**5. Test your components**
+**5. Control your components**
 
 When you configure a component, a remote control panel is generated for it in the **CONTROL** tab of the Viam app.
 With the panels, you can drive motors at different speeds, view your camera feeds, see sensor readings, and generally test the basic functionality of your machine before you've even written any code.
```

Comment:
Inside services the sentence introducing the bullets or the bullets need to change. They require a "which" or added colons. For example:

```
**Data Management**: Capture and sync data from one or more machines, and use that data for machine learning and beyond.
```